lib/data_sources/repo_docs.rb

```diff
@@ -1,60 +1,158 @@
-# TODO(ts): Rewrite data source and use one single instance to combine all
-# different versions for a given path.
-class RepoDocsDataSource < ::Nanoc::DataSources::Filesystem
+require 'uri'
+
+# The RepoDocs datasource provices items sourced from other Git repositories.
```

Comment:
```suggestion
# The RepoDocs datasource provides items sourced from other Git repositories.
```

Reply:
I should really enable the spell checker plugin more often ...